examples/posenet/helperFunc.js

```diff
@@ -0,0 +1,362 @@
+        let bias = [];
+        shape.push(manifest[layer_1]["shape"]);
+        shape.push(manifest[layer_2]["shape"]);
+        shape_bias.push(manifest[layer_1_bias]["shape"]);
+        shape_bias.push(manifest[layer_2_bias]["shape"]);
+        let filename1 = manifest[layer_1]["filename"];
+        let filename2 = manifest[layer_2]["filename"];
+        let filename1_bias = manifest[layer_1_bias]["filename"];
+        let filename2_bias = manifest[layer_2_bias]["filename"];
+        let data_1 = await util.getvariable(util.getURL(version)+filename1, true);
+        let data_2 = await util.getvariable(util.getURL(version)+filename2, true);
+        let data_1_bias = await util.getvariable(util.getURL(version)+filename1_bias, true);
+        let data_2_bias = await util.getvariable(util.getURL(version)+filename2_bias, true);
+        weights.push(new Float32Array(data_1));
+        weights.push(new Float32Array(data_2));
+        bias.push(new Float32Array(data_1_bias));
+        bias.push(new Float32Array(data_2_bias));
+        return [shape, weights, shape_bias, bias];
+    }
+}
+
+async function getOutputLayer(layername, version){
+    let util = new Utils();
+    let manifest = await util.loadmanifest(util.getURL(version));
+    let shape;
+    let shape_bias;
+    let weights, bias;
+    shape = manifest["MobilenetV1/"+layername+"_2/weights"]["shape"];
+    shape_bias = manifest["MobilenetV1/"+layername+"_2/biases"]["shape"];
+    let data = await util.getvariable(util.getURL(version)+manifest["MobilenetV1/"+layername+"_2/weights"]["filename"], true);
+    weights = new Float32Array(data);
+    let data_bias = await util.getvariable(util.getURL(version)+manifest["MobilenetV1/"+layername+"_2/biases"]["filename"], true);
+    bias = new Float32Array(data_bias);
+    return [shape, weights, shape_bias, bias];
+}
+
+//obtain desired size output
+function toOutputStridedLayers(convolutionDefinition, outputStride) {
+    let currentStride = 1;
+    let rate = 1;
+    return convolutionDefinition.map(function (_a, blockId) {
+        let convType = _a[0], stride = _a[1];
+        let layerStride, layerRate;
+        if (currentStride === outputStride) {
+            layerStride = 1;
+            layerRate = rate;
+            rate *= stride;
+        }
+        else {
+            layerStride = stride;
+            layerRate = 1;
+            currentStride *= stride;
+        }
+        return {
+            blockId: blockId, convType: convType, stride: layerStride, rate: layerRate,
+            outputStride: currentStride
+        };
+    });
+}
+
+function resize(dimension){
+    let new_dimension = [];
+    new_dimension.push(dimension[dimension.length-1])
+    for(let i =0; i<dimension.length-1; i++){
+        new_dimension.push(dimension[i]);
+    }
+    return new_dimension;
+}
+
+function transpose_weights(weights, dimension){
+    let product = dimension.reduce(function(a,b){return a*b});
+    let new_weights = new Array(product);
```

Comment:
new `Float32Array` instead of `Array` here, so you can avoid convert a array to float32 array later.